docs/scripts/generate_llmstxt.py

```diff
@@ -1,195 +1,38 @@
-        url = f"{BASE_URL}{url_path}"
 
-        md_block = f"<!--- {section}: {category} -->\n<!--- {title} -->\n\n# {title}\n\n{cleaned}\n\n[Source]({url})"
-        sections[section][category].append((title, url, md_block))
-
-        json_entries.append(
-            {
-                "title": title,
-                "url": url,
-                "section": section,
-                "category": category,
-                "content": cleaned,
-            }
-        )
+def move_digest_to_static(digest_path: Path, target_path: Path):
+    """Move digest.txt to llms.txt in the static folder."""
+    target_path.parent.mkdir(parents=True, exist_ok=True)
+    shutil.move(str(digest_path), str(target_path))
+    print(f"Moved digest to: {target_path}")
 
-    toc_lines = ["## Table of Contents", ""]
-    for section_name in ["Docs", "Optional"]:
-        for category in sorted(sections[section_name].keys()):
-            toc_lines.append(f"### {category}")
-            for title, url, _ in sections[section_name][category]:
-                toc_lines.append(f"- [{title}]({url})")
-            toc_lines.append("")
 
-    full_md = [
-        "# Weave Documentation",
-        "> Consolidated LLM-friendly documentation for Weave.",
-        "",
-        *toc_lines,
-    ]
+def main():
+    script_path = Path(__file__).resolve()
+    docs_dir = script_path.parents[1]  # weave/docs
+    static_llms_path = docs_dir / "static" / "llms.txt"
 
-    for section_name in ["Docs", "Optional"]:
-        for category in sorted(sections[section_name].keys()):
-            full_md.extend(entry for _, _, entry in sections[section_name][category])
-            full_md.append("")
-
-    txt_output.write_text("\n\n".join(full_md), encoding="utf-8")
-    json_output.write_text(json.dumps(json_entries, indent=2), encoding="utf-8")
-
-    if split_by_category:
-        for section_name in sections:
-            for category in sections[section_name]:
-                filename = f"llms-{category.lower().replace(' ', '_')}.txt"
-                category_path = txt_output.parent / filename
-                category_content = [
-                    entry for _, _, entry in sections[section_name][category]
-                ]
-                category_path.write_text(
-                    "\n\n".join(category_content), encoding="utf-8"
-                )
+    digest_file = run_gitingest_on_docs_dir(docs_dir)
+    move_digest_to_static(digest_file, static_llms_path)
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser()
-    parser.add_argument(
-        "--categories", action="store_true", help="Generate per-category .txt files"
-    )
-    args = parser.parse_args()
-
-    generate_llms_full_outputs(
-        docs_dir=Path("./docs"),
-        txt_output=Path("./static/llms-full.txt"),
-        json_output=Path("./static/llms-full.json"),
-        split_by_category=args.categories,
-    )
+    main()
```

Comment:
This Python script essentially uses shell commands. It could have been a shell script. Good as-is for now.